box/tarantool_test.go

```diff
@@ -83,7 +90,8 @@ func TestBox_Sugar_Schema_UserCreate_NoError(t *testing.T) {
 	conn, err := tarantool.Connect(ctx, dialer, tarantool.Opts{})
 	require.NoError(t, err)
 
-	b := box.New(conn)
+	b, err := box.New(conn)
+	require.NoError(t, err)
```

Comment:
It looks like we could replace box.New with box.MustNew in our tests to avoid the error check. In that case we need additional tests that New creates a workable object based on a passed connection.
```suggestion
	b, err := box.MustNew(conn)
```